### ISSUE TITLE
Could you please explain why do you split the training, validation and test set in this way?

### ISSUE THREAD

**Issue by terU3760** [tensorflow/models#4702]
Hi, Chris Shallue,

The TensorFlow framework for [im2txt model](https://github.com/tensorflow/models/tree/master/research/im2txt) about the paper: Show and Tell: A Neural Image Caption Generator is magnificent. But could you please explain why you split the training, validation and test set in such a way in your code as all images in MSCOCO training dataset and 85% of images in MSCOCO validation dataset for training; 5% of images in MSCOCO validation dataset for validation and 10% of images in MSCOCO validation dataset for test?

I'm using the model of https://github.com/tensorflow/models/tree/master/research/im2txt.
My operating system is Ubuntu 17.10.
I installed TensorFlow directly through git.
My TensorFlow version is 1.4.0.
My bazel version is 0.9.0
My CUDA version 8.0
My cuDNN  version is 6.0
My GPU model is GTX1080 with 8GB memory.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by ymodak** [comment]
@cshallue Can you please take a look? Thanks!

**Comment by cshallue** [comment]
This is what the authors of the original paper did. If I remember correctly, the test set does not have public captions, so we create our own test set from the validation set. The validation set was so large that it was not practical to evaluate over the whole thing during training, so part of it was put into the training set.
